### PR TITLE
LPD-39202 Remove role and aria-orientation

### PR DESCRIPTION
**Pull request by fortunatomaldonado**
https://liferay.atlassian.net/browse/LPD-39202
https://liferay.atlassian.net/browse/LPP-56033

As Marcos Castros said to remove the role and aria-orientation as it is not needed and causes accessibility error.
https://liferay.atlassian.net/browse/LPP-56033?focusedCommentId=2762261

Please let me know if there are any questions or comments about this.
Thank you!

### CODE REVIEW

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

**Review by markocikos — COMMENTED** [review]
@fortunatomaldonado please see comment inline

File: modules/test/playwright/tests/frontend-taglib-clay/panelGroupTag.spec.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+			await page.goto(
+				`${liferayConfig.environment.baseUrl}/web${site.friendlyUrlPath}${layout.friendlyUrlPath}`
+			);
+
+			const panelTab = page.getByRole('tablist').getByText('Panel');
+
+			await panelTab.waitFor({state: 'visible'});
+
+			const panelGroupElement = page.locator(
+				'.panel-group[role="tablist"][aria-orientation="vertical"]'
```
[the remark this answers is not in the document]
Comment (fortunatomaldonado) [reply modules/test/playwright/tests/frontend-taglib-clay/panelGroupTag.spec.ts]: Makes sense, made changes.

**Review by markocikos — APPROVED** [review]
@fortunatomaldonado LGTM! :sparkles:

**Comment by fortunatomaldonado** [comment]
ci:forward

**Comment by fortunatomaldonado** [comment]
FWD: https://github.com/brianchandotcom/liferay-portal/pull/155232

**Pull request closed without merging**